install/operator/pkg/generator/assets/infrastructure/02-syndesis-secrets.yml.tmpl

```diff
@@ -19,26 +19,24 @@
       syndesis.io/type: infrastructure
   stringData:
     OPENSHIFT_OAUTH_CLIENT_SECRET: |-
-      {{.Env.OPENSHIFT_OAUTH_CLIENT_SECRET}}
-    OPENSHIFT_OAUTH_CLIENT_SECRET: |-
-      {{.Env.OPENSHIFT_OAUTH_CLIENT_SECRET}}
+      {{.OpenShiftOauthClientSecret}}
     POSTGRESQL_PASSWORD: |-
-      {{.Env.POSTGRESQL_PASSWORD}}
+      {{.Syndesis.Components.Database.Password}}
     POSTGRESQL_SAMPLEDB_PASSWORD: |-
-      {{.Env.POSTGRESQL_SAMPLEDB_PASSWORD}}
+      {{.Syndesis.Components.Database.SampledbPassword}}
```

Comment:
Any special treatment of these attributes due to them being 'sensitive'? I originally left them in ENV due to that reason.

Reply:
They receive the same treatment. Sensitive fields are randomly generated and saved in  `syndesis-server-secret` secret. 

https://github.com/syndesisio/syndesis/pull/7114/files#diff-8f63b1158cd0c8188207de68b0013e69R221-R224

This is only changed the place where those fields are now stored after being read from the secret ... or being generated.